ckan/logic/action/create.py

```diff
@@ -724,8 +724,18 @@ def _group_or_org_create(context: Context,
     group_type = data_dict.get('type', 'organization' if is_org else 'group')
     group_plugin = lib_plugins.lookup_group_plugin(group_type)
 
-    schema: Schema = context.get(
-        "schema") or group_plugin.create_group_schema()
+    try:
+        schema: Schema = (
+            context.get("schema") or group_plugin.create_group_schema()
+        )
+    except AttributeError:
+        # TODO: remove these fallback deprecated methods in the next release
+        try:
+            schema = getattr(group_plugin, "form_to_db_schema_options")({
+                'type': 'create', 'api': 'api_version' in context,
+                'context': context})
+        except AttributeError:
+            schema = group_plugin.form_to_db_schema()
```

Comment:
these will swallow AttributeErrors from inside the function calls which could be a source of very confusing bugs. Maybe some `hasattr` checks would be better?

Reply:
Yes, absolutely. My only defense is that I just copied the code that was previously there :) See https://github.com/ckan/ckan/pull/8069/commits/dbbf0fa005ab88763f8f32204d542860a82f998b